File: src/codes/scattered-concerns/php/ksrc.php
<?php

class ApiClient {
  public function fetchData(string $endpoint): void {
    $apiKey = getenv("API_KEY");
    echo "[ApiClient] APIキー: {$apiKey}\n";
    echo "[ApiClient] エンドポイントにアクセス: {$endpoint}\n";
  }
}

class BatchJob {
  public function run(): void {
    $runMode = getenv("MODE");
    echo "[BatchJob] 実行モード: {$runMode}\n";
    echo "[BatchJob] バッチ処理開始\n";
  }
}

File: src/codes/scattered-concerns/py/ksrc.py
import os

class ApiClient:
    def fetch_data(self, endpoint: str):
        api_key = os.getenv("API_KEY")
        print(f"[ApiClient] APIキー: {api_key}")
        print(f"[ApiClient] エンドポイントにアクセス: {endpoint}")

class BatchJob:
    def run(self):
        run_mode = os.getenv("MODE")
        print(f"[BatchJob] 実行モード: {run_mode}")
        print("[BatchJob] バッチ処理開始")
